sections/footer.liquid

```diff
@@ -73,14 +73,43 @@
                       {%- endfor -%}
                     </ul>
                   {%- endif -%}
+                {%- when 'brand_information' -%}
+                  <div class="footer-block-brand-information">
+                    {%- if settings.brand_image != blank -%}
+                      {%- assign brand_image_height = settings.brand_image_width | divided_by: settings.brand_image.aspect_ratio -%}
+                      <style>
+                        .brand-information__image-wrapper {
+                          max-width: min(100%, {{ settings.brand_image_width }}px);
+                        }
+                      </style>
+                      <div class="brand-information__image-wrapper global-media-settings">
```

Comment:
This class doesn't have a parent `brand-information` one to justify the `__image-wrapper` so maybe it could just be `brand-information-image-wrapper` ? 🤷